packages/frontend/src/renderer/components/Channel/ChannelComponent.tsx

```diff
@@ -71,11 +71,15 @@ export const ChannelComponent: React.FC<ChannelComponentProps> = ({
     scrollBottom()
   }, [])
 
-  const { ref: scrollbarRef } = useResizeDetector({ onResize })
+  const { ref: scrollbarRef } = useResizeDetector<HTMLDivElement>({ onResize })
 
   const scrollBottom = () => {
     if (!scrollbarRef.current) return
-    scrollbarRef.current.scrollTop = scrollbarRef.current.scrollHeight
+    setScrollHeight(0)
+    scrollbarRef.current.scrollTo({
+      behavior: 'auto',
+      top: Math.abs(scrollbarRef.current?.clientHeight - scrollbarRef.current?.scrollHeight)
```

Comment:
Are you sure Math.abs is correct? Shouldn't it be `scrollbarRef.current?.clientHeight` and `scrollbarRef.current?.scrollHeight` both wrapped in `Math.floor`?

Reply:
I'd have checked it on Chromebook also

Reply:
yes i want to convert to positive numbers